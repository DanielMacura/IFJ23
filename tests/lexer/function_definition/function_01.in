func myFunction (_ a: Int, _ b: Int) -> Int {
    return 
}