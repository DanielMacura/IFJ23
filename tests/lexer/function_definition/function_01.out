KW_FUNC
IDENTIFIER
LBRACKET
TOKEN_UNDERSCORE
VARIABLE
COLON
KW_INT
COMMA
TOKEN_UNDERSCORE
VARIABLE
COLON
KW_INT
RBRACKET
MINUS
GT
KW_INT
LCURLYBRACKET
EOL
KW_RETURN
EOL
RCURLYBRACKET
EOF
